qlib/RestClient.qm

```diff
@@ -124,6 +124,9 @@ printf("%N\n", ans.body);
 
     @section restclientrelnotes Release Notes
 
+    @subsection restclientv1_7 RestClient v1.7
```

Comment:
this should be 1.6.1 for an intermediate release

Reply:
thanks, fixed :)